content/cloud-docs/api-docs/state-version-outputs.mdx

```diff
@@ -153,3 +153,65 @@ curl \
   }
 }
 ```
+
+## Show Current State Version Outputs for a Workspace
+
+This endpoint allows users without permissions to read state versions to fetch outputs for a workspace.
```

Comment:
Clarifying questions - does this mean that anyone with the URL can send a request to the API? No token no auth, etc, is needed?

Reply:
Nope, you will need to be a member of the organization the workspace belongs to and supply your user token. However it's for users who are a member of a team (within said organization) that doesn't have permissions to read state versions.

Reply:
Ah OK - that makes sense. What do you think about updating the sentence to reflect what you've said in your comment?

Maybe something like:

```suggestion
This endpoint allows workspace organization users who do not have permissions to read state versions to fetch outputs for a workspace.
```